src/redux/Checkout/actions.js

```diff
@@ -5,7 +5,8 @@ import {
   initStripe,
 } from '@stripe/stripe-react-native';
 import _ from 'lodash';
-import { createAction } from 'redux-actions';
+import { createAction } from '@reduxjs/toolkit';
```

Comment:
Would have been less diff to alias the new function to `createRTKAction` instead of aliasing the old one 🙂